kDriveCore/Data/DownloadQueue/DownloadQueue.swift

```diff
@@ -22,6 +22,14 @@ import InfomaniakCore
 import InfomaniakDI
 import RealmSwift
 
+// TODO: Move to core
+extension SendableDictionary {
+    var isEmpty: Bool {
+        // swiftlint:disable empty_count
+        count > 0
```

Comment:
Logic is wrong here:
```suggestion
        count == 0
```

Also when moving to core don't forget to call isEmpty instead

Reply:
Good catch. Patched.
I do not intend to change core right away since as a breaking version is in the works.